internal/helm/helmdriver/transport_http.go

```diff
@@ -212,6 +236,30 @@ func decodeUpgradeReleaseHTTPRequest(_ context.Context, r *http.Request) (interf
 	}, nil
 }
 
+func encodeUpgradeReleaseHTTPResponse(ctx context.Context, w http.ResponseWriter, response interface{}) error {
+	resp, ok := response.(UpgradeReleaseResponse)
+	if !ok {
+		return errors.NewWithDetails("failed to encode release resources response")
+	}
+
+	if resp.Err != nil {
+		return errors.NewWithDetails("failed to retrieve release resources")
+	}
+
+	// backwards compatibility!
+	oldResponse := struct {
+		ReleaseName string                 `json:"releaseName"`
+		Notes       string                 `json:"notes"`
+		Resources   []helm.ReleaseResource `json:"resources"`
+	}{
+		ReleaseName: resp.Release.ReleaseName,
+		Notes:       resp.Release.ReleaseInfo.Notes,
+		//Resources:   resp.Release.ReleaseResources,
```

Comment:
is this intentional? I mean we return this in the install response, but not here

Reply:
yes, to be similar to the. old implementation